c7n/resources/ses.py

```diff
@@ -0,0 +1,32 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+from c7n.manager import resources
+from c7n.query import DescribeSource, QueryResourceManager, TypeInfo
+from c7n.utils import local_session
+from c7n.tags import universal_augment
+
+class DescribeConfigurationSet(DescribeSource):
+
+    def augment(self, resources):
+        client = local_session(self.manager.session_factory).client('ses')
+        for r in resources:
+            details = client.describe_configuration_set(ConfigurationSetName=r['Name'], 
+                ConfigurationSetAttributeNames=['eventDestinations','trackingOptions','deliveryOptions','reputationOptions'])
```

Comment:
Do we need this at all? Looking at the api, it seems to be a required field and you're trying to feed it all possible values. I feel that we could omit this completely

Reply:
I see why it's there. Without setting `ConfigurationSetAttributeNames`, we just don't get anything back.
```
>>> response = client.describe_configuration_set(ConfigurationSetName='c7n-test')
>>> response
{'ConfigurationSet': {'Name': 'c7n-test'}, 'ResponseMetadata': {'RequestId': '73aaad6f-6c0b-4978-8521-c291e236e3f7', 'HTTPStatusCode': 200, 'HTTPHeaders': {'date': 'Wed, 12 Apr 2023 21:58:30 GMT', 'content-type': 'text/xml', 'content-length': '373', 'connection': 'keep-alive', 'x-amzn-requestid': '73aaad6f-6c0b-4978-8521-c291e236e3f7'}, 'RetryAttempts': 0}}
```

```
>>> response = client.describe_configuration_set(ConfigurationSetName='c7n-test', ConfigurationSetAttributeNames=['deliveryOptions', 'eventDestinations', 'trackingOptions', 'reputationOptions',])
>>> response
{'ConfigurationSet': {'Name': 'c7n-test'}, 'DeliveryOptions': {'TlsPolicy': 'Require'}, 'ReputationOptions': {'SendingEnabled': True, 'ReputationMetricsEnabled': True}, 'ResponseMetadata': {'RequestId': '4bcdf5ed-c74a-43af-8892-ccb2da922b4b', 'HTTPStatusCode': 200, 'HTTPHeaders': {'date': 'Wed, 12 Apr 2023 21:58:13 GMT', 'content-type': 'text/xml', 'content-length': '612', 'connection': 'keep-alive', 'x-amzn-requestid': '4bcdf5ed-c74a-43af-8892-ccb2da922b4b'}, 'RetryAttempts': 0}}
```